readthedocs/proxito/tests/test_full.py

```diff
@@ -1395,11 +1414,13 @@ def test_cache_on_private_versions_custom_domain(self):
         self.domain.save()
         self._test_cache_control_header_project(expected_value='private', host=self.domain.domain)
 
-        # HTTPS redirect respects the privacy level of the version.
-        resp = self.client.get('/en/latest/', secure=False, HTTP_HOST=self.domain.domain)
-        self.assertEqual(resp['Location'], f'https://{self.domain.domain}/en/latest/')
-        self.assertEqual(resp.headers['CDN-Cache-Control'], 'private')
-        self.assertEqual(resp.headers['Cache-Tag'], 'project,project:latest')
+        # HTTPS redirect can always be cached.
```

Comment:
Are we doing this even for 404 domains? Otherwise we're leaking the existence of the domain/project. Not super worried about it, but assume that's a small security thing.

Reply:
Not sure what you mean, the https redirect or the response headers? We already leak if a project exits or not with our 404/401responses.

Reply:
Gotcha.